src/containers/Header.js

```diff
@@ -60,6 +73,12 @@ class Header extends React.Component {
               <li>
                 <Link style={{fontSize: '16px'}}><ConsoleDetect/></Link>
               </li>
+              <li>
```

Comment:
Selector placement needs styling.
- Header (navbar navbar-default) is growing over below elements
- It should be aligned with the Zeppelin logo, help, and home button